msal/src/main/java/com/microsoft/identity/client/AccountAdapter.java

```diff
@@ -46,7 +46,7 @@
      * @return A representation of the supplied Account, as an IAccount.
      */
     @NonNull
-    static IAccount adapt(@NonNull final IAccountRecord accountIn) {
+    public static IAccount adapt(@NonNull final IAccountRecord accountIn) {
```

Comment:
If we're going to make this public, we need to move this class to an `internal` namespace